__tests__/app.test.js

```diff
@@ -454,3 +454,47 @@ test("POST: 404 with Not Found for non-existent article ID", () => {
       expect(response.body.msg).toBe("Bad Request")
     })
 })
+// 8==========
+test("should update an artcle votes with a positive value", (done) => {
+  request(app)
+    .patch("/api/articles/1")
+    .send({ inc_votes: 5 })
+    .expect(200)
+    .end((err, res) => {
+      if (err) return done(err)
+
+      // console.log(res.body.article, "< ---------reponse")
+      // console.log(res.body.article)
+      expect(res.body.article).toBeDefined()
+
+      expect(res.body.article.votes).toBeGreaterThan(0)
+      done()
+    })
+})
+test("should handle bad request when inc_votes is not an integer", (done) => {
+  request(app)
+    .patch("/api/articles/1")
+    .send({ inc_votes: "invalid" })
+    .expect(400, done)
```

Comment:
We should be sending a message as well as a status code so it should be asserted in this  and other error testing tests.